postgresql/database.go

```diff
@@ -84,79 +78,242 @@ func debugLog(query string, args []interface{}, err error, start int64, end int6
 	}
 }
 
-func init() {
+// Returns the underlying *sqlx.DB instance.
+func (s *source) Driver() interface{} {
```

Comment:
btw, an *sqlx.DB object also holds the object DB which is the embedded *sql.DB. So someone could called `.Driver().DB` to get the *sql.DB object. I realize this would break anyone's code that is dependent on it. I am usually much more in favor of making breaking changes if I feel that they belong in the long run a certain way. Because moving forward and trying to balance people's dependencies through a rolling repository of code is just crazy. People should be using some dependency management tool, there are so many, at which point deps are locked to a particular version.

I am in favor of making the breaking change and just putting a notice in the README of updates, and mark a tag / version number. But, I'm also totally cool with it if you decide to go with option b, I can still make it work comfortably in my code.